lua/ulx/modules/sh/pcrestrict.lua

```diff
@@ -0,0 +1,45 @@
+CATEGORY_NAME = "CFC"
+
+local function addToWhitelist( callingPlayer, targetPlayers )
+    local affected_plys = {}
+
+    for _, ply in pairs( targetPlayers ) do
+        local isPlyWhitelisted = isPlayerWhitelisted( ply )
+
+        if isPlyWhitelisted then 
```

Comment:
This could be redone to:

```lua
if playerIsWhitelisted( ply ) then
```
(if you renamed the function, for example)

reads just as well